Plugins/Flow.Launcher.Plugin.Folder/Main.cs

```diff
@@ -164,25 +166,15 @@ private void InitialDriverList()
         private void LoadEnvironmentStringPaths()
         {
             _envStringPaths = new Dictionary<string, string>();
-            
-            var specialPaths = 
-                new Dictionary<string,Environment.SpecialFolder> {
-                    { "appdata", Environment.SpecialFolder.ApplicationData },
-                    { "localappdata", Environment.SpecialFolder.LocalApplicationData },
-                    { "programfiles", Environment.SpecialFolder.ProgramFiles },
-                    { "programfiles(x86)", Environment.SpecialFolder.ProgramFilesX86 },
-                    { "programdata", Environment.SpecialFolder.CommonApplicationData },
-                    { "userprofile", Environment.SpecialFolder.UserProfile }
-                };
-
-            foreach (var special in specialPaths)
+
+            var specialPaths = System.Environment.GetEnvironmentVariables();
+            foreach (DictionaryEntry special in specialPaths)
```

Comment:
Nitpick: can we inline these? The `specialPaths` variable name is also misleading after this change.

Reply:
Sure thing, I've changed that in the latest commit.